packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.Styling.Example.tsx

```diff
@@ -9,18 +9,26 @@ import {
 import { mergeStyleSets, ITheme, createTheme } from 'office-ui-fabric-react/lib/Styling';
 import { Customizer } from 'office-ui-fabric-react/lib/Utilities';
 
-// Custom theme passed to Customizer
 const customThemeForShimmer: ITheme = createTheme({
   palette: {
-    // palette slot used in Shimmer for main background
```

Comment:
nit: These comments were here to let users looking at the example why specifically these palette slots were changed without a need to look into source code.

Reply:
Updated the title of the PR to better reflect the change